### ISSUE TITLE
poor performance - do you have some more tuning tipps for the docs?

### ISSUE THREAD

**Issue by ThomasWaldmann** [gnubert/katz#3]
I only get a few kB/s throughput (about 10x times less than expected). 

I first tried with defaults, then set -t 100 -q 100 on both sides (3x 33ms RTT - the docs are a bit vague and just say "multiple").

Wondering if there could be some general tipps how to determine the best values and fully use the connection speed.

**Comment by ThomasWaldmann** [comment]
It improved when I used -t 330 -q 10 (so -t is 10x RTT now).

Now getting 72KB/s which is about as good as that DSL upstream.

**Comment by gnubert** [comment]
Feel free to open a pull request to improve the documentation ;)

**Comment by ThomasWaldmann** [comment]
I don't think that having played with it once on one machine qualifies me as advice giver / doc writer.
